src/language/index.ts

```diff
@@ -0,0 +1,94 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { workspace, ExtensionContext } from 'vscode';
+
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind
+} from 'vscode-languageclient';
+
+let client: LanguageClient;
```

Comment:
Can this live on the `LanguageServerController` class?

Reply:
Done.